### PR TITLE
fix: p2p cookie issue

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by konstantinosG-deriv — APPROVED** [review]

File: src/store/region-context.tsx [hunk 1]
```diff
@@ -84,17 +84,21 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_dev = isLocalhost() || isTestlink()
         if (website_status) {
             const { clients_country, p2p_config } = website_status
+            if (is_p2p_enabled()) {
+                setP2PAllowedCountry(true)
+                setP2PLoading(false)
+            } else if ('p2p_config' in website_status && p2p_config) {
+                setP2PAllowedCountry(true)
+                setP2PLoading(false)
```
Comment (mayuran-deriv) [inline src/store/region-context.tsx:92]: hi @Nuzhy-Deriv 
here in this if & else if  both condition have same  ( setP2PAllowedCountry(true)
                setP2PLoading(false)) content possible to simplify this ?

Comment (mayuran-deriv) [inline src/store/region-context.tsx:92]:
```suggestion
            if (is_p2p_enabled() ||  ('p2p_config' in website_status && p2p_config)) {
                setP2PAllowedCountry(true)
                setP2PLoading(false)
            }  
```

**Review by hasan-deriv — APPROVED** [review]

File: src/store/region-context.tsx [hunk 2]
```diff
@@ -84,17 +84,21 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_dev = isLocalhost() || isTestlink()
         if (website_status) {
             const { clients_country, p2p_config } = website_status
+            if (is_p2p_enabled()) {
+                setP2PAllowedCountry(true)
```
Comment (aswathy-deriv) [inline src/store/region-context.tsx:88]: just a suggestion that is it possible to write the conditions inside a function and then call the function here for better readability.Can you simplify it.Thank you

File: src/store/region-context.tsx (hunk 1 above)
Comment (mayuran-deriv) [reply src/store/region-context.tsx:92]: and since every condition has   setP2PLoading(false) possible to pull out this set status out side ?

**Review by farhan-nurzi-deriv — APPROVED** [review]

**Review by mayuran-deriv — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Review by mitra-deriv — APPROVED** [review]

**Pull request merged**